<launch>
    <include file="/home/denise/catkin_ws/src/flea3/launch/plane_magnet.launch"> 
            <arg name="device" value="14434226"/>
    </include>
    <node   pkg="image_proc"
            name="image_proc"
            type="image_proc"
            ns="pg_14434226">
    </node>
    <node   pkg="image_view" 
            name = "disp"
            type = "image_view" 
            args="image:=/pg_14434226/image_rect" >
    </node>
    <node name="cal_plane_coils"
            pkg="plane_camera_magnet" 
            type = "cal_plane_coils"
            output="screen">
    </node>
</launch>
